hedera-node/src/test/java/com/hedera/services/ServicesStateTest.java

```diff
@@ -501,19 +510,26 @@ public void logsNonNullHashesFromSavedState() {
 		inOrder.verify(networkCtx).getHash();
 		inOrder.verify(book).getHash();
 		inOrder.verify(runningHashLeaf).getHash();
-		inOrder.verify(mockLog).info(argThat((String s) -> s.startsWith("[SwirldState Hashes]")));
 		inOrder.verify(ctx).update(subject);
+		// and:
+		assertThat(
+				logCaptor.infoLogs(),
+				contains(
+						equalTo("Init called on Services node 1 WITH Merkle saved state"),
+						startsWith("initial Hash in RecordsRunningHashLeaf"),
+						startsWith("[SwirldState Hashes]"),
+						startsWith("Mock for MerkleNetworkContext"),
+						equalTo("--> Context initialized accordingly on Services node 1"),
+						equalTo("ServicesState init with 0 accounts"),
+						equalTo("ServicesState init with 0 topics")));
```

Comment:
I'm not sure about the formatting / indentation here. I'm guessing it is done this way because of the way line continuation indents behave. This is a nit, not important to this fix but it is a topic I want to discuss at some point. What code style guidelines do we use and are we enforcing them via checkstyle?

Reply:
Afaik we're all just using the Oracle code style built into IntelliJ (`Preferences > Editor > Code Style > Java | Scheme: Oracle Code Style`).